wtfis/clients/abuseipdb.py

```diff
@@ -0,0 +1,43 @@
+from typing import Optional
+
+from requests.exceptions import HTTPError
+
+from wtfis.clients.base import BaseIpEnricherClient, BaseRequestsClient
+from wtfis.models.abuseipdb import AbuseIpDb, AbuseIpDbMap
+
+
+class abuseIPDBClient(BaseRequestsClient, BaseIpEnricherClient):
```

Comment:
Oops, I missed this one. Can you change to CamelCase, i.e. `AbuseIpDbClient` ?

Reply:
Done.